README.md

```diff
@@ -7,11 +7,44 @@ pnpm i -D tailwind-easy-theme colord
 ## Usage
 
 ```javascript
-const { Theme } = require('./build/index.js');
+const { Theme } = require("tailwind-easy-theme");
+const colors = require("tailwindcss/colors");
+
+const themes = {
+  light: {
+    somecolor: "#e9e6ff",
+    blue: colors.blue
+    primary: {
+      100: "#ffcccc",
+      200: "#ff9999",
+      300: "#ff6666",
+      400: "#ff3333",
+      500: "#ff0000",
+      600: "#cc0000",
+      700: "#990000",
+      800: "#660000",
+      900: "#330000"
+    },
+  },
+  dark: {
+    primary: {
+      DEFAULT: "#0f172a",
```

Comment:
This should not be allowed, since DEFAULT is not declared in the primary theme. So DEFAULT here would not have a configured tailwind class.

Primary theme creates tailwind classes and its css variables.
Variant themes create overriding css variables.